services/offset-earth/offset-earth-carbon.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { floorCount } = require('../color-formatters')
+const { BaseJsonService } = require('..')
+
+const apiSchema = Joi.object({
+  total: Joi.number()
+    .positive()
+    .required(),
```

Comment:
Should this one also be `nonNegativeInteger`?

Reply:
It's not an integer, but if there is a `nonNegativeFloat` or something I could use that?

Reply:
I don't think there is - this is probably fine as it is, if we found ourselves using this too often, we can create one then :)